features/external_sync.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Verify that syncing with a deleted user errors correctly
+
+@Integration @2.31
+Scenario: Create and restore a user, then delete and sync
+    Then I uninstall the "com.dimagi.test.external" apk app
+    Then I install the "commcare-tester-app-debug" apk
+    Then I open the "com.dimagi.test.external/com.dimagi.test.external.ExternalAppActivity" app
+    Then I touch the "ACQUIRE COMMCARE KEY" text
```

Comment:
You can't run calabash on other apps, you have to dispatch to monkeyrunner ([here for example](https://github.com/dimagi/commcare-ui-tests/blob/master/monkey/camera_image_capture.py))
The reason, afaik, is that when you run calabash on an app, it takes the apk and installs some underlying instrumentation, hence calabash functionality only works on that given apk.